taxi/urls.py

```diff
@@ -13,38 +13,64 @@
     ManufacturerCreateView,
     ManufacturerUpdateView,
     ManufacturerDeleteView,
+    DriverCreateView,
+    DriverDeleteView,
+    DriverUpdateView,
+    RemoveDriverFromCarView,
+    AssignDriverToCarView,
 )
 
 urlpatterns = [
     path("", index, name="index"),
     path(
         "manufacturers/",
         ManufacturerListView.as_view(),
-        name="manufacturer-list",
+        name="manufacturer_list",
     ),
     path(
         "manufacturers/create/",
         ManufacturerCreateView.as_view(),
-        name="manufacturer-create",
+        name="manufacturer_create",
     ),
     path(
         "manufacturers/<int:pk>/update/",
         ManufacturerUpdateView.as_view(),
-        name="manufacturer-update",
+        name="manufacturer_update",
     ),
     path(
         "manufacturers/<int:pk>/delete/",
         ManufacturerDeleteView.as_view(),
-        name="manufacturer-delete",
+        name="manufacturer_delete",
```

Comment:
Code Style issue: Use `-` to split words in URL identification parameter `name`, not the `_`. Please change `manufacturer_delete` to `manufacturer-delete`.